sefaria/model/autospell.py

```diff
@@ -36,7 +36,7 @@
 
 def normalizer(lang):
     if lang == "he":
-        return lambda x: "".join([c if c in letter_scope else unidecode(c) for c in hebrew.normalize_final_letters_in_str(x)])
+        return lambda x: "".join([c if c in letter_scope else unidecode(c) for c in hebrew.normalize_final_letters_in_str(x)]).replace('"', '״')
```

Comment:
unidecode will convert gershayim to ASCII quotes. We're already using it here. All we need to do is remove gershayim from `letter_scope`.

See this:
```python
In [3]: from unidecode import unidecode

In [4]: unidecode('״')
Out[4]: '"'

```

Reply:
Note, in your code, the standard quotation mark is gershayim and ASCII is converted to gershayim. If we use unidecode, the standard quotation mark will be ASCII.